index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>David's Portfolio - Mobile</title>
+</head>
+<body>
+    <section class="works">
```

Comment:
- [ ] Please, consider adding the previous sections: toolbar, headline - I mean you should continue building your portfolio webpage.

Reply:
Well noted It has been implemented